src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs

```diff
@@ -113,11 +113,8 @@ private void CreateUser()
         [return: MarshalAs(UnmanagedType.Bool)]
         private static partial bool LogonUser(string userName, string domain, string password, int logonType, int logonProvider, out SafeAccessTokenHandle safeAccessTokenHandle);
 
-#pragma warning disable DLLIMPORTGENANALYZER015 // Use 'LibraryImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
-        // TODO: [LibraryImportGenerator] Switch to use LibraryImport once we add support for non-blittable struct marshalling.
-        [DllImport("netapi32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
-        internal static extern uint NetUserAdd([MarshalAs(UnmanagedType.LPWStr)]string servername, uint level, ref USER_INFO_1 buf, out uint parm_err);
-#pragma warning restore DLLIMPORTGENANALYZER015 // Use 'LibraryImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
+        [LibraryImport("netapi32.dll", SetLastError = true)]
```

Comment:
Wait so this can marshal p/invoke signatures with ``Span<char>`` as well?

Reply:
Only if when the [struct marshalling](https://github.com/dotnet/runtime/blob/main/docs/design/libraries/LibraryImportGenerator/StructMarshalling.md) (with the [support for collections](https://github.com/dotnet/runtime/blob/main/docs/design/libraries/LibraryImportGenerator/SpanMarshallers.md#design-2-out-line-default-support-with-extensions-to-native-type-marshalling-for-contiguous-collections)) is explicitly specified.

We do have an example of how a marshaller could be implemented for Spans: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.InteropServices/tests/Ancillary.Interop/SpanMarshallers.cs

But it is currently not integrated or exposed.